### DISCUSSION TITLE
Support highlighting typed letters in dropdown results

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#1711] [Feature requests]
It is a common business requirement to have dropdown results with highlighted letters like seen in the screenshot below.

It would be cool to have this supported for e.g. `Autocomplete` and `MultiSelect`, etc out of the box.

![Screenshot from 2022-07-01 14-48-18](https://user-images.githubusercontent.com/16797721/176897630-fe430dfb-5f23-48b6-b0c8-e6cff5756ba5.png)

**Comment by rtivital** [comment]
You can add this with Highlight component and custom item component – https://mantine.dev/core/autocomplete/#custom-item-component
